lib/compiler/src/sys_core_fold.erl

```diff
@@ -2108,7 +2108,7 @@ make_var(A) ->
 make_var_name() ->
     N = get(new_var_num),
     put(new_var_num, N+1),
-    list_to_atom("fol"++integer_to_list(N)).
+    list_to_atom("$fo"++integer_to_list(N)).
```

Comment:
This is a matter of taste, but I would prefer the prefix to be either `$fol` or `$f`. The same goes for the other prefixes. (There is no rule, as far as I know, that says that the variable prefix needs to be exactly three characters.)